### PR TITLE
Add pr merge docs

### PR DESCRIPTION
**Pull request by AgustinRamiroDiaz**
After talking with @teresaromero we found that we had different criteria for merging PRs. 

This PR is an attempt to write down how we'd like out usual PR-merging to be, and also to get more feedback on the different views of the team

### CODE REVIEW

File: docs/code-review.md
Comment (jLopezbarb) [inline docs/code-review.md:68]: Can we add what happen if the author is an external contributor? They are not allowed to merge the PR

Comment (AgustinRamiroDiaz) [reply docs/code-review.md:68]: Sure! How do you think we should handle that situation?

I'm thinking that  
- If there are maintainers among the reviewer, they should merge the pr as they have better context
- If there are not maintainers among the reviewers, they would need to wait for a maintainer to take responsibility. They could ping the maintainers to remind them of the pr

Comment (jLopezbarb) [reply docs/code-review.md:68]: Maintainers are the only ones that can merge PRs. What I would do is:

- when all the requests from the maintainers have been completed(all the maintainers that have written something accepts the PR), merge
- I would ask for two approvals when an external contributor makes a PR, just to ensure that the PR is aligned with what we want

**Review by jLopezbarb — APPROVED** [review]

File: docs/code-review.md
Comment (jLopezbarb) [inline docs/code-review.md:77]:
```suggestion
- All requests from the maintainers have been addressed, and those maintainers have approved the PR.
```

File: docs/code-review.md
Comment (jLopezbarb) [inline docs/code-review.md:72]:
```suggestion
#### **If the author is not a maintainer**
```

File: docs/code-review.md
Comment (jLopezbarb) [inline docs/code-review.md:68]:
```suggestion
#### **If the author is a maintainer**
```

**Review by ifbyol — APPROVED** [review]

**Comment by jLopezbarb** [comment]
Check pre-commit linter before merging but LGTM!

**Pull request merged**
